var/spack/repos/builtin/packages/py-cuml/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCuml(PythonPackage):
+    """cuML is a suite of libraries that implement machine
+    learning algorithms and mathematical primitives functions
+    that share compatible APIs with other RAPIDS projects."""
+
+    homepage = "https://github.com/rapidsai/cuml"
+    url      = "https://github.com/rapidsai/cuml/archive/v0.15.0.tar.gz"
+
+    version('0.15.0',  sha256='b6b37c0f370cd4e881fc24083166ee86a934f1b823159ad36fac6457412c79cd')
+
+    depends_on('python@3.7:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-cython', type='build')
+    depends_on('py-numpy', type=('build', 'run'))
+    depends_on('py-numba', type=('build', 'run'))
+    depends_on('py-cudf', type=('build', 'run'))
+    depends_on('cuda')
+    depends_on('py-cupy', type=('build', 'run'))
+    depends_on('py-scipy', type=('build', 'run'))
+    depends_on('treelite+python', type=('build', 'run'))
+    depends_on('py-joblib', type=('build', 'run'))
+    depends_on('py-scikit-learn', type=('build', 'run'))
```

Comment:
I don't see a lot of these deps in the `setup.py`, where did you find these?

Reply:
They show up as missing when you go to try to `import cuml`.